src/game/game.jsx

```diff
@@ -0,0 +1,304 @@
+import { Controls } from "./controls/controls.jsx";
+import { Menu } from "./menu/menu.jsx";
+import { Score } from './score/score.jsx';
+import s from "./game.module.css";
+
+export class Game extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = getFromLocalStorage('state') ?
+			this.state = getFromLocalStorage('state') :
+			this.state = {
+				history: [
+				{
+					squares: Array(9).fill(null)
+				}
+				],
+				stepNumber: 0,
+				xIsNext: true,
+				single: false,
+				isMenuOpen: true,
+				score: [0, 0],
+			};
+		this.cleanHistory = this.cleanHistory.bind(this);
+		this.jumpTo = this.jumpTo.bind(this);
+		this.changeMode = this.changeMode.bind(this);
+		this.toggleMenu = this.toggleMenu.bind(this);
+		this.changeScore = this.changeScore.bind(this);
+		this.resetScore = this.resetScore.bind(this);
+	}
+
+	handleClick(i) {
+		const history = this.state.history.slice(0, this.state.stepNumber + 1);
+		const current = history[history.length - 1];
+		const squares = current.squares.slice();
+
+		if (calculateWinner(squares) || squares[i]) {
+			return;
+		}
+
+		squares[i] = this.state.xIsNext ? "X" : "O";
+		this.setState({
+			history: history.concat([
+			{
+				squares: squares
+			}
+			]),
+			stepNumber: history.length,
+			xIsNext: !this.state.xIsNext,
+			score: (calculateWinner(squares) ? this.changeScore(calculateWinner(squares) === 'X' ? 0 : 1) : (this.state.score)),
+		});
+	}
+
+	jumpTo(step) {
+		if (this.state.single === true) return;
+		this.setState({
+			stepNumber: step,
+			xIsNext: step % 2 === 0
+		});
+	}
+
+	cleanHistory() {
+		this.setState({
+			history: [
+			{
+				squares: Array(9).fill(null)
+			}
+			],
+			stepNumber: 0,
+			xIsNext: true,
+		});
+	}
+
+	changeMode() {
+		this.setState({
+			single: !this.state.single,
+		})
+		this.resetScore();
+	}
+
+	toggleMenu() {
+		this.setState({
+			isMenuOpen: !this.state.isMenuOpen,
+		});
+	}
+
+	changeScore(index) {
+		return this.state.score.map((item, ind) => {
+			if(ind === index) return ++item;
+			return item;
+		});
+	}
+
+	resetScore() {
+		this.cleanHistory();
+		this.setState({
+			score: [0,0],
+		});
+	}
+
+	render() {
+		setToLocalStorage('state', this.state);
+		const history = this.state.history;
+		const current = history[this.state.stepNumber];
+		const winner = calculateWinner(current.squares);
+		
+		let status;
+		if (winner) {
+			status = (
+			<p> 
+				{ winner === 'X' ?
+					(<span style={{ color: "red" }}> Player X</span>) :
+					(<span style={{ color: "blue" }}> Player O</span>)
+				}
+				<span style={{ color: "green" }}> WINS!</span>
+			</p>
+			);
+		} else if (!current.squares.includes(null)) {
+			status = (
+			<p>
+				<span style={{ color: "orange" }}>DRAW!</span>
+			</p>
+			);
+		} else {
+			status = (
+			<p>
+				Next move:
+				{this.state.xIsNext ? (
+				<span style={{ color: "red" }}> Player X</span>
+				) : (
+				<span style={{ color: "blue" }}> Player O</span>
+				)}
+			</p>
```

Comment:
restructure it to avoid repetetive code

Reply:
Ok! Should it look like this?

```suggestion
        <span style={this.state.xIsNext ? { color: "red" }: {color: "blue"}}> {this.state.xIsNext ? 'Player X' : 'Player Y'} </span>
```

Reply:
`
<span style={{ color: this.state.xIsNext  ? "red" :  "blue"}}> {`Player ${this.state.xIsNext ? X : Y}`} </span>
`